src/Viewer.js

```diff
@@ -125,11 +134,13 @@ export class Viewer {
             this.controls.maxPolarAngle = Math.PI * .75;
             this.controls.minPolarAngle = 0.1;
             this.controls.enableDamping = true;
-            this.controls.dampingFactor = 0.05;
+            this.controls.dampingFactor = 0.25;
```

Comment:
Any particular reason for this change? If the editor needs higher damping, maybe we can make it a parameter to the `Viewer`.

Reply:
When the damping is a little lower the "demand" rendering has a much quicker effect. I little of a personal preference, but I also think it's easier to work with in an edit mode, when you want more control of the angles. With that said, this change should maybe be inside the Editor class if we decide to keep it.

Reply:
Regarding if we should make it a parameter to the Viewer class I think we can keep it as it is and let the users reach into the viewer to modify the controls and camera. In my experience it's pretty common practice when working with three. Maybe we could mark them as public for clarity.

Reply:
Agreed that this should be set from within the `Editor` class. Maybe a parameter to the `Viewer` class doesn't quite make sense. Also, we should be careful about creating too many parameters for `Viewer`